test/strong_routes/allow_test.rb

```diff
@@ -48,18 +48,29 @@
     end
 
     it "allows access to /users?stuff=12" do
-      get "/users/?stuff=12"
+      get "/users?stuff=12"
```

Comment:
The original test has an odd `/` before the `?` which I am assuming was a typo, but it still passing until this change. If we need to allow this. let me know.